pkg/plugins/golang/v3/scaffolds/init.go

```diff
@@ -124,12 +109,8 @@ func (s *initScaffolder) Scaffold() error {
 			KustomizeVersion:         KustomizeVersion,
 			ControllerRuntimeVersion: ControllerRuntimeVersion,
 		},
+		&kdefault.Kustomization{},
```

Comment:
This was changed to be ordered by package, so can you keep this where it was previously instead of moving it to the midle of the `templates` package templates?